FogCreek/Glitch-Community
# Cassey-community: accessibility improvements to pagination control

## Links
* [Remix link](cassey-community.glitch.me)
* Manuscript - [Focus should go to previous button after clicking last possible 'next' button](https://glitch.manuscript.com/f/cases/3328645/Focus-is-lost-when-you-click-the-next-button-in-a-paginated-project-list-and-arrive-at-the-last-page) and [announcement for screenreader users that page content changed when they pressed pagination buttons](https://glitch.manuscript.com/f/cases/3328694/Screenreader-users-should-hear-an-announcement-that-clicking-the-pagination-buttons-caused-the-visible-projects-to-change)

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/4480480/58837558-841c4a00-8621-11e9-8814-de73ec46b42e.png)

## Changes:
* Adds a space between 'Show all' and 'badge' so screenreader users hear 'show all X' instead of 'show allx', removes some margin to account for this. Didn't see anywhere else this would be visually affected. 

## How To Test:
* Paginate to the end of your recent projects list with the mouse or keyboard; focus should return to the "Previous" button when you get to the end of the list. If running voiceover or another screenreader, you should hear 'Page x of y' on every page after you press the button. You should also hear 'Showing all projects' after you click the 'show all' button.

## Feedback I'm looking for:
- Reducer use look correct?
